torch/__init__.py

```diff
@@ -911,6 +911,63 @@ def is_warn_always_enabled():
     """
     return _C._get_warnAlways()
 
+
+def _check_with(error_type, cond, *args):
```

Comment:
I'm not really sure what utility we get from slavishly following the C++ convention. In Python, the idiomatic convention would be to just `check(cond, lambda: f"blah {v} {x}")`; which, btw, is the helper we have in PrimTorch that we'd eventually want to dedupe these with.

Reply:
Just to be clear, you're suggesting changing `args` to just be a single argument instead of an argument list, right? That sounds good to me